v1/fleet_units/exhibitor.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Exhibitor/Zookeeper
+After=docker.service
+Requires=docker.service
+
+[Service]
+Environment=EXHIBITOR_IMAGE=mbabineau/zookeeper-exhibitor:3.4.6_1.5.4
```

Comment:
they haven't built that in master yet:
![image](https://cloud.githubusercontent.com/assets/2601345/8734102/c04dcd8c-2bd5-11e5-99eb-4cdf874e206a.png)